src/Data/List/Base.agda

```diff
@@ -428,9 +428,10 @@ linesBy {A = A} P? = go nothing where
 
   go : Maybe (List A) → List A → List (List A)
   go acc []       = maybe′ ([_] ∘′ reverse) [] acc
-  go acc (c ∷ cs) with does (P? c)
-  ... | true  = reverse (Maybe.fromMaybe [] acc) ∷ go nothing cs
-  ... | false = go (just (c ∷ Maybe.fromMaybe [] acc)) cs
+  go acc (c ∷ cs) = let acc′ = Maybe.fromMaybe [] acc in
```

Comment:
Any chance we could lay this out as:
```
go acc (c ∷ cs) =
  let acc′ = Maybe.fromMaybe [] acc in
  if does (P? c)
    then reverse acc′ ∷ go nothing cs
    else go (just (c ∷ acc′)) cs
```

Reply:
Will do (eventually)

Reply:
Done.

Reply:
Apologies, I realise I didn't do *precisely* what you asked for, and then re-read #2369 so have updated to use a `where` clause now... and now the layout is consistent with `wordsBy` in L452-L454 below...